### PR TITLE
batchnorm module

### PR DESCRIPTION
**Pull request by Flowingsun007**

### CODE REVIEW

File: oneflow/python/nn/modules/bachnorm.py [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+        - Input: :math:`(N, C)` or :math:`(N, C, L)`
+        - Output: :math:`(N, C)` or :math:`(N, C, L)` (same shape as input)
+
+    Examples::
+
+        >>> # With Learnable Parameters
+        >>> m = nn.BatchNorm1d(100)
+        >>> # Without Learnable Parameters
+        >>> m = nn.BatchNorm1d(100, affine=False)
+        >>> input = torch.randn(20, 100)
```
Comment (MARD1NO) [inline oneflow/python/nn/modules/bachnorm.py:229]: 文档里出现了torch

Comment (Ldpe2G) [reply oneflow/python/nn/modules/bachnorm.py:234]: 后面再refine吧，现在是直接抄过来的

**Review by Ldpe2G — APPROVED** [review]
